roles/etcd/handlers/main.yml

```diff
@@ -17,19 +19,12 @@
   command: systemctl daemon-reload
   when: ansible_service_mgr == "systemd"
 
-- name: start etcd
-  service:
-    name: etcd
-    enabled: yes
-    state: started
-  when: is_etcd_master
-
-- name: start etcd-proxy
-  service:
-    name: etcd-proxy
-    enabled: yes
-    state: started
-  when: is_etcd_proxy
+- name: wait for etcd up
+  uri: url=http://localhost:2379/health
```

Comment:
please better off use the etcd_endpoint fact, or I can fix that later, after the https://github.com/kubespray/kargo/pull/367